transport/rif_comms/client.py

```diff
@@ -8,95 +8,101 @@
     Msg,
     RskAddress,
     Void,
-    Subscriber,
-    BooleanResponse
+    Subscriber
 )
 from transport.rif_comms.proto.api_pb2_grpc import CommunicationsApiStub
 
 
-class RifCommsClient:
+class Client:
     """
     Class to connect and operate against a RIF Communications pub-sub node.
     """
 
     def __init__(self, rsk_address: Address, grpc_api_endpoint: str):
         """
         Constructs the RIF Communications Client.
-        :param rsk_address: address of the node that wants to use the RIF Communications server
+        :param rsk_address: RSK address of the node that wants to use the RIF Communications server
         :param grpc_api_endpoint: GRPC URI of the RIF Communications pub-sub node
         """
         self.rsk_address = RskAddress(address=rsk_address)
-        self.grpc_channel = insecure_channel(grpc_api_endpoint)
+        self.grpc_channel = insecure_channel(grpc_api_endpoint)  # TODO: how to make this secure?
```

Comment:
We might take a look at this https://grpc.io/docs/guides/auth/

i have created this issue for the future https://jirainfuy.atlassian.net/browse/RLP-896

Reply:
thanks :+1: